pkgs/development/tools/rust/cbindgen/default.nix

```diff
@@ -1,23 +1,32 @@
-{ lib, stdenv, fetchFromGitHub, rustPlatform, Security }:
+{ lib, stdenv, fetchFromGitHub, rustPlatform, python3Packages, Security }:
 
 rustPlatform.buildRustPackage rec {
   pname = "rust-cbindgen";
-  version = "0.15.0";
+  version = "0.16.0";
```

Comment:
Is this intended? Your commit message says 0.17.0.

Reply:
Nope, I apparently forgot to add a local change I have lying around. Will fix asap. Thanks for spotting this.